packer/scripts/install-certstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit -o xtrace
+
+# Certstrap is used for creating the k8s signing keys, which are necessary to avoid communication bugs on host restarts
+echo "Installing certstrap ..."
+# We run chown in docker to avoid requiring sudo
+systemctl start docker.service
```

Comment:
Only the golang image should be downloaded at this point. I thought about cleaning up, but the scripts which run after `vagrant up` use this same image.. I found that `packer build`ing with the image cleaned up vs. without shaves off about 200 MB. However, this is at least 200 MB which would otherwise have to be downloaded on every vagrant up, so I believe the one-time overhead for each vagrantbox outweighs the overhead which cleanup introduces on every `vagrant up` otherwise.